Mathlib/MeasureTheory/Function/StronglyMeasurable/Basic.lean

```diff
@@ -1905,6 +1905,10 @@ theorem _root_.aestronglyMeasurable_withDensity_iff {E : Type*} [NormedAddCommGr
     simpa only [NNReal.coe_eq_zero, Ne.def] using h'x
 #align ae_strongly_measurable_with_density_iff aestronglyMeasurable_withDensity_iff
 
+lemma of_absolutelyContinuous (h : ν ≪ μ) (hμ : AEStronglyMeasurable g μ) :
+    AEStronglyMeasurable g ν := let ⟨g₁, hg₁, hg₁'⟩ := hμ; ⟨g₁, hg₁, h.ae_eq hg₁'⟩
```

Comment:
Loogle says this already exists as [mono'](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Function/StronglyMeasurable/Basic.html#MeasureTheory.AEStronglyMeasurable.mono%27). We can add an `alias` though.

Reply:
`mono_ac` is the name I was looking for. I renamed it now.